### ISSUE TITLE
Colors are not interpreted correctly

### ISSUE THREAD

**Issue by SidneyNemzer** [jsakas/CSSStyleDeclaration#48]
**Keywords**
CSS has hundreds of color keywords, but CSSStyle only recognizes a few
https://github.com/chad3814/CSSStyleDeclaration/blob/master/lib/parsers.js#L100-L116

**HSL and HSLA**
CSSStyle has no logic to interpret HSL or HSLA values. Here's the regex for Hex, RGA, and RGBA. HSL and HSLA are missing.

https://github.com/chad3814/CSSStyleDeclaration/blob/master/lib/parsers.js#L28-L30

**Computing the result**
Hex values are correctly converted to RGB, however keywords (even valid ones) are not. If CSSStyle intends to emulate a browser's behavior, it should convert everything to RGB or RGBA

**TLDR**
CSSStyle doesn't handle color values correctly

**Comment by FagnerMartinsBrack** [comment]
This is causing [an issue](https://github.com/tmpvar/jsdom/issues/1965#issuecomment-326177053) with JSDOM.

**Comment by PavelPZ** [comment]
... and all its dependencies, as JEST, [REACT-TESTING-LIBRARY](https://github.com/kentcdodds/react-testing-library/issues/214) etc.
Is PR welcomed? Can I fix [parsers.js](https://github.com/jsakas/CSSStyleDeclaration/blob/master/lib/parsers.js#L100-L116)?

**Comment by bennypowers** [comment]
This breaks lighthouse tests:

<img width="877" alt="screenshot 2019-02-02 22 01 08" src="https://user-images.githubusercontent.com/1466420/52168724-0fe69e00-2736-11e9-9e6c-8ed7c9f6e9bd.png">
<img width="320" alt="screenshot 2019-02-02 22 00 57" src="https://user-images.githubusercontent.com/1466420/52168725-0fe69e00-2736-11e9-9888-f69754b8055e.png">
